server/src/dtos/sync.dto.ts

```diff
@@ -27,7 +27,9 @@ export class AssetFullSyncDto {
 export class AssetDeltaSyncDto {
   @ValidateDate()
   updatedAfter!: Date;
+
   @ValidateUUID({ each: true })
+  @ApiProperty({ name: 'userIds[]' })
```

Comment:
You'd want `@IsArray()` instead. That also gives you proper validation

Reply:
Are you sure? I think, I tried that first. IsArray is already set in the open API json description.

This change makes the client send a single value so that express sees it as an array in any case.

Reply:
Apparently you are right, sorry. I think you want both annotations then though, right?

Reply:
You should be able to delete the API property without impacting the spec file

Reply:
After you delete this line, feel free to merge it.

Reply:
This was specifically for a query param, which indeed was needed. I just changed it to post to avoid these types of serialization issues entirely.